docker/release.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-vt_base_version='v13.0.0'
+vt_base_version='v16.0.0'
```

Comment:
Is this step of updating the base-version something we should change in the `do-release` script automatically? cc - @frouioui

Reply:
that's a good idea.
Or we should include in the post-release PR where we update the versions in a bunch of files (like pom.xml).

Reply:
Yep. I'll talk to @frouioui and do a follow-up PR for this change.

Reply:
I created an issue for this at #11681 and a PR with the changes to accomplish this at #11682